plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/internal/editor/tabbar/TabbarToolBarManager.java

```diff
@@ -60,8 +57,7 @@ public class TabbarToolBarManager extends ToolBarManager {
     public TabbarToolBarManager(ToolBar toolBar, IWorkbenchPart part) {
         super(toolBar);
         this.tabbarPart = part;
-        this.onResize = event -> updateGradientBackground();
-        getControl().addListener(SWT.Resize, onResize);
+        getControl().addListener(SWT.Resize, event -> updateGradientBackground());
```

Comment:
Add a comment to explain that the listener will be automatically removed. This allows to clarify why there is no "getControl().removeListener" later.

Reply:
Done